spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -446,6 +446,7 @@ def _get_ring_buffer_to_input_left_shifts(
         weights_signed = False
         rate_stats = [RunningStats() for _ in range(n_synapse_types)]
         steps_per_second = 1000000.0 / machine_timestep
+        min_max_weight = numpy.ones(n_synapse_types) * 2 ** 32
```

Comment:
Note that this stores the "minimum of the max weights" rather than the "minimum weight".  This is because the connector code does not include this.  This could be added, but it should be noted that:
 - This would have to be the minimum absolute weight as the maximum is the maximum absolute weight
 - This would have to account for the fact that the minimum might be 0, which is always representable!  Thus it should be the minimum non-zero weight.